topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -520,7 +543,7 @@
    ],
    "source": [
     "connection.execute(sa.text(\"\"\"\n",
-    "CREATE TABLE IF NOT EXISTS \"doc\".\"weather_data\" (\n",
+    "CREATE TABLE IF NOT EXISTS \"notebook\".\"weather_data\" (\n",
```

Comment:
Maybe let's remove this explicit reference, so that software tests can use a different schema. The other notebooks also don't include it, right?

Reply:
yes

Reply:
Erledigt.